Lib/asyncio/unix_events.py

```diff
@@ -51,8 +51,14 @@ def __init__(self, selector=None):
 
     def close(self):
         super().close()
-        for sig in list(self._signal_handlers):
-            self.remove_signal_handler(sig)
+        if not sys.is_finalizing():
```

Comment:
What happens if someone calls `loop.close()` twice?  I guess `self._signal_handlers.clear()` is needed.

Reply:
Double warning will be shown obviously.
But it is very rare situation: most likely the loop will be closed by `__del__`.
Even `atexit` callbacks are called when `sys.is_finalizing()` is `False`.

Reply:
Well, in this case I'd like to clear the list of signal handlers.

Reply:
Ok, will make a postfix PR soon.